src/sagemaker_containers/worker.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import collections
+
+from flask import Flask, Response
+
+import sagemaker_containers as smc
+
+env = smc.environment.ServingEnvironment()
+
+
+def default_healthcheck_fn():  # type: () -> Response
+    """Ping is default health-check handler. Returns 200 with no content.
+
+    During a new serving container startup, Amazon SageMaker starts sending periodic GET requests to the /ping endpoint
+    to ensure that the container is ready for predictions.
+
+    The simplest requirement on the container is to respond with an HTTP 200 status code and an empty body. This
+    indicates to Amazon SageMaker that the container is ready to accept inference requests at the /invocations endpoint.
+
+    If the container does not begin to consistently respond with 200s during the first 30 seconds after startup,
+    the CreateEndPoint and UpdateEndpoint APIs will fail.
+
+    While the minimum bar is for the container to return a static 200, a container developer can use this functionality
+    to perform deeper checks. The request timeout on /ping attempts is 2 seconds.
+
+    More information on how health-check works can be found here:
+    https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html#your-algorithms-inference-algo-ping-requests
+
+    Returns:
+        (flask.Response): with status code 200
+    """
+    return Response(status=smc.status_codes.OK)
+
+
+def run(transform_fn, initialize_fn=None, healthcheck_fn=None, module_name=None):
```

Comment:
What about just doing healthcheck_fn=default_healthcheck_fn? Then the default is documented in the function signature rather than in the code, and you can avoid all assignments to `healthcheck_fn or default_healthcheck_fn`